flocker/restapi/_infrastructure.py

```diff
@@ -165,6 +166,9 @@ def structured(inputSchema, outputSchema, schema_store=None):
     :param schema_store: A mapping between schema paths
         (e.g. ``b/v1/types.json``) and the JSON schema structure, allowing
         input/output schemas to just be references.
+    :param ignore_body: If true, ignore the contents of the body for all
```

Comment:
Maybe expand on what "ignore" means here.  The body is still read.  It just isn't decoded.  Maybe `decode_body=True` is closer to the intent?